### ISSUE TITLE
random = 1 leads to different results for the same runs

### ISSUE THREAD

**Issue by ghadahamed** [AlexeyAB/darknet#4919]
When I training the pretrained darknet on my custom data but with enabling the random parameter, this leads to different results using the same config file and all settings are the same.
The mAP varies within 65% to 80% which is considered wide range for the same data and configuratins, is this actually because the random flag sometimes resizes to small sizes and leads to low results and sometimes resizes to large sizes and consequently gets better results or there is something else?
can @AlexeyAB guide me in this please?

**Comment by AlexeyAB** [comment]
mAP will be caclulated by using initial network size (width= height= in cfg file) during training, regardless of random=1 or 0.

**Comment by ghadahamed** [comment]
Thanks for you @AlexeyAB for your reply. Really I were confused for this so thanks for your reply, it is really appreciated for me.
Sorry for disturbing you, but kindly can you help me in discovering what can cause this?
Say I run the first time and got specific results. I deleted all the backup weight files and re-run again, different results are obtained .. what can cause this @AlexeyAB ?

Sorry again for disturbing you and thanks a lot for your reply and help.

**Comment by AlexeyAB** [comment]
Very low training/validation datset.

Or you are doing something wrong.

**Comment by ghadahamed** [comment]
Yes, my dataset is small, all the dataset is 116 and I divided it into 80% and 20% and no validation.
So is this the reason and is there anything you advise me to do in such cases?
